### ISSUE TITLE
Add __radd__() to EnvPath

### ISSUE THREAD

**Issue by astronouth7303** [xonsh/xonsh#2446]
```
TypeError: can only concatenate list (not "EnvPath") to list 
```

Add `.__radd__()` to `EnvPath` to make manipulating `$PATH` easier.

**Comment by astronouth7303** [comment]
Actually, add an `.__add__()`, too.

**Comment by melund** [comment]
Just for reference: https://gitter.im/xonsh/xonsh?at=596fb2c7bc4647297406fafd

**Comment by scopatz** [comment]
Thanks for reporting! See #2451

**Comment by melund** [comment]
This is fixed in #2451
